src/cascadia/TerminalApp/ActionAndArgs.cpp

```diff
@@ -33,6 +29,12 @@ static constexpr std::string_view ResizePaneKey{ "resizePane" };
 static constexpr std::string_view MoveFocusKey{ "moveFocus" };
 static constexpr std::string_view FindKey{ "find" };
 static constexpr std::string_view ToggleFullscreenKey{ "toggleFullscreen" };
+static constexpr std::string_view ToggleCommandPaletteKey{ "toggleCommandPalette" };
```

Comment:
```suggestion
static constexpr std::string_view ToggleCommandPaletteKey{ "commandPalette" };
```
Genuinely forgot if/how we resolved this in the spec. So, sorry if you have to repeat yourself.

Reply:
You know, I like this a lot more. Plus, this will make a lot more sense with `{ "action": "commandPalette", "mode":"comamndline" }`